Cargo.lock

```diff
@@ -15,6 +15,7 @@ dependencies = [
  "async-trait 0.1.41 (registry+https://github.com/rust-lang/crates.io-index)",
  "bytes 0.5.6 (registry+https://github.com/rust-lang/crates.io-index)",
  "hex-literal 0.3.1 (registry+https://github.com/rust-lang/crates.io-index)",
+ "ion-c-sys 0.1.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Is there a newer version of ion-c-sys? https://docs.rs/ion-c-sys/0.4.3/ion_c_sys/index.html

Reply:
Yeah. I haven't been running `cargo update`. I can do that for this PR, but just note I haven't done any testing with the newer version. Your choice!

Reply:
Ran `cargo updated` and it didn't. Ran `upgrade` and it did!